client/js/screens/Event/Event.js

```diff
@@ -28,113 +27,123 @@ const COMMENT_MUTATION = gql`
               </Text>
-              <Text style={styles.text}>{startDay}</Text>
-            </>
-          ) : (
-            <Text style={styles.text}>
-              {startTime}, {startDay} to {endTime} {endDay}
-            </Text>
-          )}
-        </View>
-        <View style={styles.buttonContainer}>
-          <CalendarButton event={event} />
-        </View>
-        <View style={styles.detailsContainer}>
-          <Image style={styles.image} source={eventImage} />
-          <Text style={styles.halfSpaceText}>{event.description}</Text>
-        </View>
-        <View style={styles.inputContainer}>
-          <Mutation
-            mutation={COMMENT_MUTATION}
-            client={
-              new ApolloClient({
-                uri: APOLLO_SERVER_ADDRESS,
-              })
-            }>
-            {updateEvent => {
-              return (
-                <Form
-                  onSubmit={async values => {
-                    try {
-                      const updatedEvent = await updateEvent({
-                        variables: {
-                          title: event.title,
-                          comment: values.comment,
-                          username: user.name,
-                        },
-                      });
-                    } catch (e) {
-                      this.setState({error: e});
-                    }
-                  }}
-                  render={({handleSubmit}) => (
-                    <>
-                      <Field
-                        name="comment"
-                        render={({input, meta}) => (
-                          <TextInput
-                            style={styles.input}
-                            id="comment"
-                            placeholder="Comment"
-                            placeholderTextColor="black"
-                            type="text"
-                            inputProps={{
-                              autoComplete: 'off',
-                            }}
-                            {...input}
-                          />
-                        )}
-                      />
-                      <TouchableOpacity onPress={handleSubmit}>
-                        <Image
-                          style={styles.commentIcon}
-                          source={commentIcon}
+            )}
+          </View>
+          <View style={styles.buttonContainer}>
+            <CalendarButton event={event} />
+          </View>
+          <View style={styles.detailsContainer}>
+            <Image style={styles.image} source={{uri: event.image}} />
+            <Text style={styles.halfSpaceText}>{event.description}</Text>
+          </View>
+          <View style={styles.inputContainer}>
+            <Mutation
+              mutation={COMMENT_MUTATION}
+              client={
+                new ApolloClient({
+                  uri: APOLLO_SERVER_ADDRESS,
+                })
+              }>
```

Comment:
No need to specify the client for this mutation. It should automatically be your APOLLO_SERVER_ADDRESS.